### ISSUE TITLE
Next steps for improvements of all demo progress bars

### ISSUE THREAD

**Issue by CodyCBakerPhD** [catalystneuro/tqdm_publisher#54]
Very relevant to how this should ultimately appear on the GUIDE...

Can you add 
- text to the left of each bar drawing from the `desc` of the `format_dict`?
- text counter of `n` and `total` as `f"{n}/{total} ({n/total:.2f}%)"` - can either be inside the bar or to the right of the bar, or whoever else you think would be visually appealing
- estimate of speed per update and time remaining (`remaining` and `rate_fmt` of the `format_dict` should be helpful for this)

Basically trying to map as much as possible between a classic console TQDM and the much prettier front-end one
